webapp/src/components/dot_menu.tsx

```diff
@@ -114,6 +117,7 @@ const DotMenu = (props: DotMenuProps) => {
         <MenuButton
             title={props.title}
             ref={rootRef}
+            isActive={(props.isActive ?? false) || isOpen}
```

Comment:
Isn't this the same as
```suggestion
            isActive={props.isActive ?? isOpen}
```

Reply:
It seems these conditions are not the same. i.e. if `props.isActive` is `false` and `isOpen` is `true`, first condition result is `true`, and second is `false`.

Reply:
Aaaah, that's true, because the ?? only checks for real `null`ity, right?

Reply:
Yep